packages/@dcl/ecs/tools/protocol-buffer-generation/index.ts

```diff
@@ -15,7 +15,6 @@ const NON_EXPOSED_LIST = [
   1090, // Billboard
   1050, // UiTransform
   1070, // AvatarModifierArea,
-  1071, // CameraModeArea
```

Comment:
run make build. We have a bug that doesnt check if the `.md` changes on build.
So by now we have to always run `make build` so the latests changes gets commited

Reply:
Done!